cmake/mkl/MKLConfig.cmake

```diff
@@ -661,126 +758,112 @@ foreach(lib ${MKL_LIBRARIES})
 endforeach()
 
 # Threading selection
-if(MKL_THREADING)
+if(MKL_THREADING STREQUAL "tbb_thread" OR MKL_DPCPP_THREADING STREQUAL "tbb_thread")
+  find_package(TBB CONFIG COMPONENTS tbb )
   if(MKL_THREADING STREQUAL "tbb_thread")
-    find_package(TBB CONFIG COMPONENTS tbb )
-    if(TARGET TBB::tbb)
-      set(MKL_THREAD_LIB $<TARGET_LINKER_FILE:TBB::tbb>)
-      set(MKL_SDL_THREAD_ENV "TBB")
-      get_property(TBB_LIB TARGET TBB::tbb PROPERTY IMPORTED_LOCATION_RELEASE)
-      get_filename_component(TBB_LIB_DIR ${TBB_LIB} DIRECTORY)
+    set(MKL_THREAD_LIB $<TARGET_LINKER_FILE:TBB::tbb>)
```

Comment:
The TBB part is not aligned with opensource because at some point I added manual search for TBB to opensource in case cmake file is not found (some packages has only tbb libraries without cmake file). I guess we should add this search to Intel oneMKL instead of removing it from here.

Reply:
Thank you for letting me know!
I added missed part for tbb